scripts/datasets/light_microscopy/check_hpa.py

```diff
@@ -10,7 +10,14 @@
 def check_hpa():
     from util import ROOT
 
-    loader = get_hpa_segmentation_loader(os.path.join(ROOT, "hpa"), "train", (512, 512), 1, download=True)
+    loader = get_hpa_segmentation_loader(
+        path=os.path.join(ROOT, "hpa"),
+        split="train",
+        patch_shape=(1024, 1024),
+        batch_size=1,
+        channels=["protein"],
```

Comment:
Have you checked that this works when using multiple channels?

Reply:
Yes, it works as expected!